events/generate/data/firebase-db1-cloudevent-input.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
In general this CloudEvent form is never produced by Eventarc, so it's not an accurate conformance test. Eventarc produces binary CloudEvents, so you'l see all of the CloudEvent metadata in HTTP headers.

Not sure how we generated this.

Even if a FF passes the conformance test for a structured CloudEvent, that doesn't guarantee actual conformance.

Reply:
This is merely the test data representation -- how else would you propose to represent it? We create a CloudEvent from this, and send it via the Go SDK, which defaults to sending the binary encoding (https://github.com/cloudevents/sdk-go/blob/master/v2/binding/write.go#L86).

Reply:
This is fine and more ideal in terms of a format than a binary representation.

It's just a bit confusing how this is in the `events/generate` folder.